tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -479,7 +479,7 @@ $(1): $$($(1)_BINARY)
 $(1)_bin: $$($(1)_BINARY).bin
 test_$(1): $$($(1)_BINARY)
 	@test -f $$(TEST_SCRIPT) || (echo 'Unable to find the test script. Is the software emulation available in $$(TARGET)?'; exit 1)
-	$$(TEST_SCRIPT) $$($(1)_BINARY) '~~~ALL TESTS PASSED~~~'
+	$$(TEST_SCRIPT) $$($(1)_BINARY) $$(TEST_PASS_STRING) $$(TARGET)
```

Comment:
Looks like we're weren't passing TARGET in before. I'm confused how this worked previously.

Reply:
It worked for everything but running tests with renode (which is the only place where we need `TARGET`.

Even with renode, it works when we call `make test` which is what is part of our CI since that uses a different test rule which does pass in the target:
https://github.com/tensorflow/tensorflow/blob/d9841dfd9689f9c4e0bc4e1229dbc354f01ebc1b/tensorflow/lite/micro/tools/make/targets/bluepill_makefile.inc#L69-L71

This common code in helper_functions.inc is only used when we do:
```
make -f tensorflow/lite/micro/tools/make/Makefile TARGET=bluepill test_kernel_add_test
```

which is what this change is fixing.